changelog.d/20240405_151230_fahad.khalid.md

```diff
@@ -0,0 +1 @@
+- [Improvement] Update User Activity dataset query to improve average time spent in course. (by @Fahadkhalid210)
```

Comment:
nit: we can add context here on why this change provides a better data then before.